packages/react/src/components/ReadOnlyValue/_read-only-value.scss

```diff
@@ -2,7 +2,8 @@
   display: block;
   margin-bottom: $spacing-06;
   label,
-  textarea {
+  textarea,
+  div {
```

Comment:
this only works if they pass in a div. Should we wrap the node content in a div to ensure this style is applied?

Reply:
Actually, i'm wrapping the value prop into the div, at the code... check line 44 in `ReadOnlyValue.jsx`

Reply:
You are right. My mistake!